docs/how-tos/integrating-with-webflow.md

```diff
@@ -0,0 +1,102 @@
+---
+sidebar_position: 7
+---
+
+# Integrating Eppo with Webflow
+
+
+In this tutorial, you will learn how how run landing page tests in Webflow after minimal  engineering setup. At a high level, you will add a client side version of Eppo’s Javascript SDK as a script in your Webflow workspace that will:
+
+- initialize the Eppo JS client
+- handle experiment assignment logic
+- dynamically update pre-defined content variables such as text and image URLs
+
+## Prerequisites:
+
+- Eppo Account
+- Webflow account
+- Client-side analytics platform like Segment, Amplitude, Adobe, Mixpanel, etc to capture Eppo assignment events and send them to your data warehouse for analysis.
+
+## Setting up Eppo
+
+Set up an feature flag in Eppo — for step by step instructions, follow our [quickstart](/feature-flags/). We will be using JSON flag type to fill in the content on our Webflow landing page. For the sake of this tutorial, make sure to include the following keys for both variations:
+
+```json
+{
+	"title": "some words",
+	"image": "https://myimage.com",
+	"subtitle": "some more words",
+	"cta": "even more words"
+}
+```
+
+You can make the values of the keys whatever makes sense for your use case, or you can use our example from our prototype below:
```

Comment:
Another thing to note is that Webflow has its own CDN. So if you want your image to be something you uploaded to Webflow you need to get its full URL by clicking on it's settings icon and then "Copy Asset Link"

<img width="249" alt="image" src="https://github.com/Eppo-exp/eppo-docs/assets/417605/d38f0e99-52b5-435b-8165-af9c978c0174">